pandas/_libs/window/aggregations.pyx

```diff
@@ -630,6 +710,18 @@ def roll_kurt(ndarray[float64_t] values, ndarray[int64_t] start,
         start, end
     )
     output = np.empty(N, dtype=float)
+    min_val = np.nanmin(values)
+
+    with nogil:
+        for i in range(0, N):
+            val = values[i]
+            if notnan(val):
+                nobs_mean += 1
+                sum_val += val
+    mean_val = sum_val / nobs_mean
```

Comment:
umm, why are you adding this?

Reply:
The issues mentioned above describes a use case. where the series is shifted by 5000. In this case, if we do not subtract the mean, The decimal places will be so small, that they will be ignored. Leading to the numerical imprecision described in the issue.

Kahan summation only solves issues, when the range of the values between the windows is really big. Could post a more precise and explicit example, if this helps

Reply:
i think i understand the issue, but am unclear why you are not *always* just subtracting the mean rather than conditionally.

Reply:
Aaah sorry. Misunderstood you.

In case of something like ``[1e12, 0.023, 0.04565, 0.343545, 0.343434]`` and subtract the mean from the complete series, we lose precision for the second window starting at ``0.023``, because this abs value gets really big too